covid_graphs/create_dev_venv.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
I think this is the same as
```suggestion
#!/usr/bin/env bash -eu
set -o pipefail
```

I've been using `bash -e` and I'm not familiar with `-u`. Perhaps even `-o pipefail` can be put there, e.g. `!/usr/bin/env bash -euo pipefail`

Reply:
I took inspiration from our codebase :) This is what "the internets" say: https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
I pipefail can't go there, so I've kept it as it was.